projects/core/src/datetimepicker/calendar.scss

```diff
@@ -42,17 +37,21 @@ $mat-calendar-next-icon-transform: translateX(-2px) rotate(45deg);
 .mat-datetimepicker-calendar-header-date-time {
   font-size: 30px;
   line-height: 34px;
-
   [mode='landscape'] & {
     white-space: normal;
     word-wrap: break-word;
   }
 }
 
+.mat-datetimepicker-calendar-header-ampm-container {
+  font-size: 23px;
```

Comment:
hard-coded size in pixels is not adaptable to screen size. what was the reason for this? could you maybe use `em`?

Reply:
Done